include/Core/Request/NonceChecker.php

```diff
@@ -58,15 +58,15 @@ public function check_nonce() {
 				$action = sanitize_text_field( $_REQUEST['action2'] );
 			}
 
-			if ( strpos( $action, 'el-log-list-' ) !== 0 ) {
+			if ( strpos( $action, 'el-' ) !== 0 ) {
 				return;
 			}
 
-			if ( ! isset( $_REQUEST[ LogListPage::LOG_LIST_ACTION_NONCE_FIELD ] ) ) {
+			if ( ! isset( $_REQUEST[ $action . '-nonce-field' ] ) ) {
```

Comment:
See https://github.com/sudar/email-log/blob/dev/2.4.3/include/Core/UI/Page/LogListPage.php#L30

The action already has `nonce` in it. So append only `-field` and not `-nonce-field`.

You would require to make the change in the add-on as well.

Reply:
@sudar is going to look into this. So assigning back to him.